src/hist/core.py

```diff
@@ -66,13 +66,20 @@ def project(self, *args: Union[int, str]):
                 f"Only projections by indices and names are supported for {self.__class__.__name__}"
             )
 
-    def plot(self, *args, **kwargs,) -> Union[Plot1D_RetType, Plot2D_RetType]:
+    def density(self):
+        """
+        Density Histogram.
+        """
+
+        return self / self.sum()  # ToDo: maybe should not be filled
```

Comment:
See https://boost-histogram.readthedocs.io/en/latest/usage/quickstart.html#computing-with-histograms

Also, overflow bins need to be removed. Let's drop this for now and try it in a new PR.

Reply:
Okay, but luckily, when I use `self.view() / self.sum() / np.prod(self.axes.widths, axis=0)`, the flow bins are removed. Not sure why, but it meets your requirement :)

Reply:
That's because `.view(flow=False)` removes the flow bins, but it also returns a NumPy array, so now there are no flow bins, but also no axes, plot functions, etc.

Reply:
Use the np array to create a new Histogram? Or just leave it to the next PR.

Reply:
Let's work on it in a new PR. We can a) make a density class - but a lot might be duplicated if we are not careful, or b) make a new histogram, removing flow bins, then input the np array above.

Reply:
Okay, so what else should I do in this PR?

Reply:
I think that's it.